### PR TITLE
Fix up handling of image data as well as setting Linux Capabilties correctly

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

**Comment by rhatdan** [comment]
@mheon @baude @jwhonce This will fix a bunch of failures, since we are not getting data out of the images.  Also should fix test/e2e/run_security_labels.go tests.

File: libpod/image/image.go [hunk 1]
```diff
@@ -847,22 +847,72 @@ func (i *Image) Dangling() bool {
...
+	return imgInspect.Env
+}
+
 // Labels returns the image's labels
-func (i *Image) Labels(ctx context.Context) (map[string]string, error) {
+func (i *Image) Labels(ctx context.Context) map[string]string {
 	imgInspect, err := i.imageInspectInfo(ctx)
 	if err != nil {
-		return nil, nil
+		return nil
```
Comment (mheon) [inline libpod/image/image.go:908]: Why are we just discarding errors? That doesn't seem sane

Comment (rhatdan) [reply libpod/image/image.go:908]: The previous code was ignoring the errors. I thought there was a reason.

Comment (mheon) [reply libpod/image/image.go:908]: Probably not? Can't think of one

Comment (rhatdan) [reply libpod/image/image.go]: Fixed

**Comment by rhatdan** [comment]
@baude @mheon @jwhonce @TomSweeneyRedHat @QiWang19 @giuseppe Lets get this in so we can pass more tests.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by rhatdan** [comment]
Also fixes run_seccomp.go e2e test.

**Comment by rhatdan** [comment]
run_selinux_test.go also passes.

**Pull request merged**